app/github/middleware.py

```diff
@@ -32,9 +39,15 @@ def process_response(self, request, response):
         token = request.session.get('access_token')
         expiration = request.session.get('access_token_last_validated')
         handle = request.session.get('handle')
+        ip_address = '24.210.224.38' if settings.DEBUG else get_real_ip(request)
```

Comment:
note: we are behidn a load balancer and cloudfront in prod.. so we'll need to make sure `get_real_ip()` works with that

Reply:
@owocki As long as nginx is setup to forward the IP addresses appropriately, we'll be fine.  I'll take a look at the nginx config and find out.